app/controllers/idv/socure/document_capture_controller.rb

```diff
@@ -0,0 +1,113 @@
+# frozen_string_literal: true
+
+module Idv
+  module Socure
+    class DocumentCaptureController < ApplicationController
+      include Idv::AvailabilityConcern
+      include IdvStepConcern
+      include DocumentCaptureConcern
+
+      before_action :confirm_not_rate_limited
+      before_action :confirm_step_allowed
+
+      def show
+        Funnel::DocAuth::RegisterStep.new(current_user.id, sp_session[:issuer]).
+          call('socure_document_capture', :view, true)
+
+        # document request
+        document_request = DocAuth::Socure::Requests::DocumentRequest.new(
+          document_capture_session_uuid: document_capture_session_uuid,
+          redirect_url: idv_socure_document_capture_url,
+          language: I18n.locale,
+        )
+
+        document_response = document_request.fetch
+
+        @document_request = document_request
+        @document_response = document_response
+        @url = document_response.dig('data', 'url')
+
+        document_capture_session = DocumentCaptureSession.find_by(
+          uuid: document_capture_session_uuid,
+        )
+
+        document_capture_session.socure_docv_token = document_response.dig(
+          'data',
+          'docvTransactionToken',
+        )
+        document_capture_session.save
+
+        # useful for analytics
+        @msg = document_response['msg']
+        @reference_id = document_response.dig('referenceId')
+      end
+
+      def update
+        clear_future_steps!
+        idv_session.redo_document_capture = nil # done with this redo
+
+        # fetch result probably not needed local dev
+        if (socure_document_uuid = request.params[:document_uuid])
+          uploaded_documents_decision(socure_document_uuid)
```

Comment:
does this #update action work? i am unable to locate the definiton of `#uploaded_documents_decision` 🤔

handling the the redirect is covered in [LG-14007](https://cm-jira.usa.gov/browse/LG-14007) so we could just remove the `#update` action